### DISCUSSION TITLE
Userscript vs External Update

### DISCUSSION THREAD

**Discussion by Malix-Labs** [Tampermonkey/tampermonkey#1707] [Q&A] [answered]
Hello!

What is the difference between userscript vs external updates?

![image](https://user-images.githubusercontent.com/76160668/218763034-e2a3131f-66d3-410c-9dab-7b79a6ebacbb.png)

**Comment by 7nik** [comment, marked as answer]
Externals are `@require`'s and `@resource`'s data

**Reply by Malix-Labs** [reply]
Thanks!

**Comment by derjanb** [comment]
Thanks for the question. See #1712
